.env

```diff
@@ -0,0 +1 @@
+VITE_FORESQUARE_API_KEY="fsq3o6S9Y+8mKA+7ZthdL/PFM6LGb0/IZlhwc8sZewKrfJQ="
```

Comment:
how come the api key committed? I thought .env is in .gitignore

Reply:
Thank you for letting me know.
It was dangerous.